mosip/on-prem/README.md

```diff
@@ -161,8 +161,14 @@ kubectl apply -f https://rancher.e2e.mosip.net/v3/import/pdmkx6b4xxtpcd699gzwdtt
 * Wait for few seconds after executing the command for the cluster to get verified.
 * Your cluster is now added to the rancher management server.
 
-## Longhorn
-* Install [Longhorn](../../longhorn/README.md) for persistent storage.
+## The following storage classes can be used:
```

Comment:
The heading cannot be a sentence.... You can have ## Storage Class and then write the sentence- The following storage classes can be used: